modules/statusline/lualine/lualine.nix

```
@@ -185,19 +185,29 @@ in {
           {
             {
               -- Lsp server name .
+
               function()
-                local msg = 'No Active Lsp'
                 local buf_ft = vim.api.nvim_buf_get_option(0, 'filetype')
```

`nvim_buf_get_option()` is deprecated. Use `nvim_get_option_value()` instead.
```suggestion
                local buf_ft = vim.api.nvim_get_option_value('filetype', {})
```

---

Good catch.